src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/index.test.js

```diff
@@ -323,4 +324,64 @@ describe('convertToOptimoBlocks', () => {
       assetType,
     });
   });
+
+  it('should render include blocks with correct index positions', async () => {
```

Comment:
```suggestion
  it('should render include blocks with correct index positions and omit non-supported/invalid includes', async () => {
```
Just to make clear why there are indexes missing in resulting block data